### PR TITLE
Update Rule “record-a-quick-and-dirty-done-video/rule”

### PR DESCRIPTION
**Pull request by jontygardner**
Automatically generated by Netlify CMS

### CODE REVIEW

**Review by adamcogan — CHANGES_REQUESTED** [review]
@jontygardner too much is lost... and not generic 
-a

File: rules/record-a-quick-and-dirty-done-video/rule.md [hunk 1]
```diff
@@ -33,17 +33,15 @@ When deciding whether a PBI might be a good contender to record a done video for
 ::: greybox
 **Choosing software to record your screen and camera together:**
 
-* Remote recordings: Zoom, or Microsoft Teams via NDI 
-* Basic recordings: Snagit, Loom
-* Advanced recordings: Camtasia
+* Remote recordings: [Zoom](https://zoom.us/), or [Microsoft Teams](https://www.microsoft.com/en-au/microsoft-teams) via [NDI,](https://www.ndi.tv/) [StreamYard](https://streamyard.com/) (advanced)
```
Comment (pierssinclairssw) [inline rules/record-a-quick-and-dirty-done-video/rule.md:36]: the word "or" is very confusing here

Comment (jontygardner) [reply rules/record-a-quick-and-dirty-done-video/rule.md]: Done - removed "or"

**Review by pierssinclairssw — APPROVED** [review]

**Review by drwharris — APPROVED** [review]

**Comment by tiagov8** [comment]
Hey @jontygardner (editor) / @adamcogan (reviewer)

This PR is here for a while... do you need help getting it done?

**Pull request merged**